modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using CLI
+
+You can configure the `NotificationsConfiguration` CR by using CLI.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance.
+
+.Procedure
+
+. Enable notifications for the Argo CD instance in the CLI:
```

Comment:
Per our [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command), steps that introduce commands should have phrasing that explicitly mentions running a command:
```suggestion
. Enable notifications for the Argo CD instance by running the following command:
```

Reply:
Fixed.